test/unit/llm-events/openai/error.test.js

```diff
@@ -18,7 +18,8 @@ tap.test('LlmErrorMessage', (t) => {
     'error.code': 'insufficient_quota',
     'error.param': 'test-param',
     'completion_id': undefined,
-    'embedding_id': undefined
+    'embedding_id': undefined,
+    'vector_store_id': undefined
```

Comment:
we should really move this test, not now but didn't realize it was still here. maybe you can move it when you add tool_id